File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for i in range(len(commands)) :
        
        s = commands[i][0]
        e = commands[i][1]
        p = commands[i][2]
        sliced_array = array[s-1:e]
        sliced_array.sort();
        pick_array = sliced_array[p-1]
        answer.append(pick_array);
        
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    for(let i = 0 ;i<commands.length;i++){
        let s = commands[i][0]-1
        let e = commands[i][1]
        let idx = commands[i][2]-1
        arr = array.slice(s,e)
        arr.sort((a,b) => a - b)
        answer.push(arr[idx])
    }
    return answer;
}